### DISCUSSION TITLE
Is evolution blind to the future or is it bound to the past?

### DISCUSSION THREAD

**Discussion by josePereiro** [josePereiro/TankeFactory#10] [Discussions]
From the selfish gene (Why are people? chapter):

> This line of thought can be put into vaguely Darwinian terms. Evolution works by natural selection, and natural selection means the differential survival of the ‘fittest’. But are we talking about the fittest individuals, the fittest races, the fittest species, or what? For some purposes this does not greatly matter, but when we are talking about altruism it is obviously crucial. If it is species that are competing in what Darwin called the struggle for existence, the individual seems best regarded as a pawn in the game, to be sacrified when the greater interest of the species as a whole requires it. To put it in a slightly more respectable way, a group, such as a species or a population within a species, whose individual members are prepared to sacrifice themselves for the welfare of the group, may be less likely to go extinct than a rival group whose individual members place their own selfish interests first. Therefore the world becomes populated mainly by groups consisting of self-sacrificing individuals. This is the theory of ‘group selection’, long assumed to be true by biologists not familiar with the details of evolutionary theory, brought out into the open in a famous book by V. C. Wynne-Edwards, and popularized by Robert Ardrey in The Social Contract. The orthodox alternative is normally called ‘individual selection’, although I per- sonally prefer to speak of gene selection. 

> The quick answer of the ‘individual selectionist’ to the argument just put might go something like this. Even in the group of altruists, there will almost certainly be a dissenting minority who refuse to make any sacrifice. It there is just one selfish rebel, prepared to exploit the altruism of the rest, then he, by definition, is more likely than they are to survive and have children. Each of these children will tend to inherit his selfish traits. After several generations of this natural selection, the ‘altruistic group’ will be over-run by selfish individuals, and will be indistinguishable from the selfish group. Even if we grant the improbable chance existence initially of pure altruistic groups without any rebels, it is very difficult to see what is to stop selfish individuals migrating in from neighbouring selfish groups, and, by inter-marriage, contaminating the purity of the altruistic groups. 

> The individual-selectionist would admit that groups do indeed die out, and that whether or not a group goes extinct may be influenced by the behaviour of the individuals in that group. He might even admit that if only the individuals in a group had the gift of foresight they could see that in the long run their own best interests lay in restraining their selfish greed, to prevent the destruction of the whole group. How many times must this have been said in recent years to the working people of Britain? But group extinction is a slow process compared with the rapid cut and thrust of individual competition. Even while the group is going slowly and inexorably downhill, selfish individuals prosper in the short term at the expense of altruists. The citizens of Britain may or may not be blessed with foresight, but evolution is blind to the future.

The apparent contradiction arises: Does the observation of long-term evolutionary strategies imply the absence of shorter-term strategies with higher fitness? If the answer is negative, what are the reasons why the shorter-term, higher-fitness strategy does not initially dominate the population? Does this suggest a relationship (constraint) between the timescales of both strategies? A diminishing return case?

**Comment by ta-rekt** [comment]
He has great examples that explain this in detail all through the book, I encourage you to keep reading. He says the definition of an evolutionarily stable (ES) strategy is one that can't be invaded by any other once it is established. And one of the main themes of the book is that there are cooperative (i.e. beneficial in the long term but not the short term) strategies that are evolutionarily stable in certain environments. Meaning even if a short-term "exploiter" appears, they will initially grow but not to the point of replacing the old species, and eventually die out or remain as a stable minority. I don't think it has to do with timescales fundamentally, although we observe a difference in timescales as a side effect. It sounds more like it's about diminishing returns of the invader.

One chapter devoted to this explains the paper that I sent you, "the evolution of cooperation", and I just saw that veritasium made a video about it!

https://www.youtube.com/watch?v=mScpHTIi-kM
